src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    static final Random random = new Random();
+
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        Figure circle = new Circle();
+        Figure rectangle = new Rectangle();
+        Figure rightTriangle = new RightTriangle();
+        Figure square = new Square();
+        Figure isoscelesTrapezoid = new Figure();
+        Figure[] figures = new Figure[]{circle, rectangle, rightTriangle, square, isoscelesTrapezoid};
+        int count = 1;
+        FigureSupplier figure = new FigureSupplier();
```

Comment:
According to the checklist, you should not create a new `FigureSupplier` inside the loop. However, there's no loop here that creates multiple `FigureSupplier` instances, but the variable name 'figure' is misleading as it represents a `FigureSupplier`. Consider renaming it to 'figureSupplier' for better clarity.